tests/functional/workloads/app/test_jenkins_simulation.py

```
@@ -79,25 +82,44 @@ def test_git_clone(self, pod, interface_iterate):
             assert (
                 relevant_pod_logs
             ), f"None of {func_calls} were not found on {cephfsplugin_pod.name} pod logs"
+            logger.info("Validated CSI function calls present in logs")
+
+            logger.test_step("Validate logs do not contain error messages")
+            logger.assertion(
+                f"Error message check: error_msg='{error_msg}', "
+                f"present={error_msg in relevant_pod_logs}"
+            )
             assert not (
                 error_msg in relevant_pod_logs
```

Ruff (0.15.15)

[error] Test for membership should be `not in`

Convert to `not in`

(E713)